test/db/analysis/golang

```diff
@@ -0,0 +1,233 @@
+0x0026c090 str.cannot_use__wallet_with__regtest__btcwallet_not_yet_compatible_with_regtest
+0x0026c520 str.Use__s__h_to_show_options
+0x0026c5f4 str.btcwallet.conf
+0x0026c688 str.Error_creating_a_default_config_file:__v
+0x0026cae4 str.s:_Multiple_network_params_can_t_be_used_together____choose_one
+0x0026cbd4 str.Error_parsing_config_file:__v
+0x0026ccb8 str.btcd.conf
+0x0026d0a8 str.m___s_rpcuser_____s
+0x0026d0fc str.m___s_rpcpass_____s
+0x0026d150 str.m___s_notls__0_1___:_s
+0x0026d2e0 str.rpcuser__s_rpcpass__s
+0x0026d368 str.notls__s
+0x0026d900 str.http
+0x0026d908 str.https
+0x0026d90c str.http
+0x0026d928 str.:
+0x0026d9c4 str.POST
+0x0026da28 str.Content_Type
+0x0026dbe8 str.error_reading_json_reply:__v
+0x0026dcec str.d__s
+0x0026dd70 str.s
+0x0026df84 str.d._d._d
+0x0026dfc0 str.beta
+0x0026e0a0 str.s__s
+0x0026e1e0 str.0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz
+0x0026e370 str.linux
+0x0026e384 str.btcd
+0x0026e3d0 str.linux
+0x0026e3e4 str.btcctl
+0x0026e430 str.linux
+0x0026e444 str.btcwallet
+0x0026e4b8 str.btcctl.conf
+0x0026e53c str.rpc.cert
+0x0026e5c0 str.rpc.cert
+14828
+EOF
+RUN
+
+NAME=Resolve all symbols on a stripped mach0 go1.18 binary (arm64)
+FILE=bins/golang/hello.mach0.stripped
+CMDS=<<EOF
+aang
+fl~sym.go.~?
+pdfs @ sym.go.runtime.panicwrap ~str.
+izq~?
+iI~compiler
+EOF
+EXPECT=<<EOF
+1429
+0x100006dd8 str.value_method
+0x100006e48 str.called_using_nil
+0x100006e64 str.pointer
+13
+compiler go1.18 (cmd -compiler=gc CGO_ENABLED=1 CGO_CFLAGS= CGO_CPPFLAGS= CGO_CXXFLAGS= CGO_LDFLAGS= GOARCH=arm64 GOOS=darwin)
+EOF
+RUN
+
+NAME=Resolve all symbols on a stripped PE go1.12 binary (x64)
+FILE=bins/golang/hello_go_strip.exe
+CMDS=<<EOF
+aang
+fl~sym.go.~?
+pdfs @ main ~str.
+izq~?
+iI~compiler
+EOF
+EXPECT=<<EOF
+1788
+0x004a699b str.hello__hacktivity
+9032
+compiler go1.15
+EOF
+RUN
+
+NAME=Parse Golang 1.18 BuildInfo ELF
```

Comment:
These normally go to ABI tests (like the ada one's)

Reply:
true, but since the bins are big, it take a bit of time to split them for these 2 tests, so i preferred to have them within one loaded bin.